documentation/commands.md

```diff
@@ -1,42 +1,49 @@
-## Command Reference
+# Command Reference
 
 Although the tutorial gives a large amount of information, if you find
 remembering the commands too complicated, here's a reference. Remember
 that you can type out multiple commands in one go; for example:
 `BAW231 fh 090 d 30 sp 180` will work as well as all three commands run
-separately. Some commands have "alises" that are shorter to type. An
+separately. Some commands have "aliases" that are shorter to type. An
 example of that would be the `takeoff` command which has an alias `to`.
 
-### Taxi
-_Aliases -_ `taxi` / `wait` / `w`
-
-_Information -_ This command tells the specified plane to taxi to a
-runway; if a runway is not included they will continue to the runway
-with the largest headwind.
-
-_Syntax -_ `AAL123 taxi [Runway]`
-
-### SID
-_Aliases -_ `sid`
-
-_Information -_ This command tells the specified plane a standard
-instrument departure route (SID) it should follow. Each SID is a list of
-fixes to be flown in sequence. Having a standardized route often helps
-organize departing traffic, and maintain separation from arriving aircraft.
-
-_Syntax -_ `AAL123 sid [SID name]`
-
-
-### STAR
-_Aliases -_ `star`
-
-_Information -_ This command tells the plane to add or change their filed
-Standard Terminal Arrival Route to match the route you specify. This must be
-entered in dotted format, and include the point where the STAR is joined, as
-well as the destination airport, for example: `MLP.GLASR9.KSEA`. See the section
-on rerouting for further detail.
-
-_Syntax -_ `AAL123 star [transition].[STAR name].[airport]`
+### Table of Contents
+
+[Departure commands](#departure-commands)
+- [Cleared as Filed](#cleared-as-filed)
+- [Climb via SID](#climb-via-sid)
+- [SID](#sid)
+- [Takeoff](#takeoff)
+- [Taxi](#taxi)
+
+[Arrival commands](#arrival-commands)
+- [Descend via STAR](#descend-via-star)
+- [Land](#land)
+- [STAR](#star)
+
+[Routing commands](#routing-commands)
+- [~~Fix~~](#fix)
+- [Hold](#hold)
+- [Proceed Direct](#proceed-direct)
+- [Route](#route)
+- [Reroute](#reroute)
+- [Say Route](#say-route)
+[Aircraft Control commands](#aircraft-control-commands)
+- [Abort](#abort)
```

Comment:
Please scratch this `abort` command out as you did with `fix` and mark it as deprecated as well. It does very little, and should be refactored into individual commands anyway (such as `go around / hold position / fly present heading` etc). Added #639 for removal of the abort command.